Moo/play/lib.py

```diff
@@ -51,46 +51,46 @@
 } 
 
 
-def albums(base, index):
-    '''
-    returns dict of minimal album metadata
-    '''
-    out = dict()
-    dupes = dict()
-
-    for ind,path in enumerate(index):
-        pobj = Path(path)
-        tracks = len([x for x in pobj.iterdir()])
-
-        try:
-            meta = metadata(base, pobj, single=True)['01']
-        except KeyError:
-            meta = {'path': path, 'tracks': tracks}
-
-        year = meta['year'][:4] if meta.get('year') else 'None'
-        artist = meta.get('album_artist') or meta.get('artist')
-
-        pruned = {
-            'album': meta.get('album'),
-            'artist': artist.split(';')[0] if artist else 'None',
-            'encoding': meta.get('encoding'),
-            'genre': meta.get('genre') or 'None',
-            'index': ind,
-            'mtime': meta.get('mtime'),
-            'tracks': tracks,
-            'year': year}
-
-        if path in out:
-            if path not in dupes:
-                dupes[path] = list()
-            dupes[path].append(path)
-        else:
-            out[path] = pruned
-
-    if dupes:
-        raise ValueError('Found duplicates: {}'.format(dupes))
-
-    return out
+# def albums(base, index):
+#     ''' Removing/Erasing
+#     returns dict of minimal album metadata
+#     '''
+#     out = dict()
+#     dupes = dict()
+
+#     for ind,path in enumerate(index):
+#         pobj = Path(path)
+#         tracks = len([x for x in pobj.iterdir()])
+
+#         try:
+#             meta = metadata(base, pobj, single=True)['01']
+#         except KeyError:
+#             meta = {'path': path, 'tracks': tracks}
+
+#         year = meta['year'][:4] if meta.get('year') else 'None'
+#         artist = meta.get('album_artist') or meta.get('artist')
+
+#         pruned = {
+#             'album': meta.get('album'),
+#             'artist': artist.split(';')[0] if artist else 'None',
+#             'encoding': meta.get('encoding'),
+#             'genre': meta.get('genre') or 'None',
+#             'index': ind,
+#             'mtime': meta.get('mtime'),
+#             'tracks': tracks,
+#             'year': year}
+
+#         if path in out:
+#             if path not in dupes:
+#                 dupes[path] = list()
+#             dupes[path].append(path)
+#         else:
+#             out[path] = pruned
+
+#     if dupes:
+#         raise ValueError('Found duplicates: {}'.format(dupes))
+
+#     return out
```

Comment:
You commented out lib.albums() (note the "s" on the end—sorry, that's confusing), but we want to leave that function the way it is—so you can comment it back in.